test/unit/test_autoflush.py

```diff
@@ -29,11 +29,11 @@ class TestAutoflush:
     def test_flushed(self, mock_obj):
         with Autoflush(mock_obj):
             pass
-        assert mock_obj.flush.called
+        assert mock_obj.flush.assert_called
```

Comment:
```suggestion
        mock_obj.flush.assert_called()
```

If I understand the `mock` documentation correctly, these asserts are functions that should be called in tests

Reply:
I think you're right